app/controllers/organized/ambassador_dashboards_controller.rb

```diff
@@ -11,10 +11,14 @@ def show
           .includes(:ambassador_task_assignments)
           .sort_by { |ambassador| -ambassador.percent_complete }
 
-      @ambassador_task_assignments =
-        current_user
-          .ambassador_task_assignments
-          .task_ordered
+      if current_user.ambassador?
+        current_ambassador = Ambassador.find(current_user.id).decorate
+        @suggested_activities = current_ambassador.suggested_activities.decorate
```

Comment:
Could you put `task_ordered` scope on this?

Reply:
It's set in the decorator:
```rb
# app/decorators/ambassador_decorator.rb L5-10 (7839d064)

def ambassador_task_assignments
  ambassador
    .ambassador_task_assignments
    .includes(:ambassador_task)
    .task_ordered
end
```